### PR TITLE
electron: upgrade to 1.8.2, node 8.2.1, chromium 59

### PR DESCRIPTION
**Pull request by brian-mann**

### CODE REVIEW

**Comment by bahmutov** [comment]
🤗

**Comment by jennifer-shehane** [comment]
Just as a note, I get this error in the console when building with this version update:

> "Electron Helper[1584:20104] Couldn't set selectedTextBackgroundColor from default ()"

This is a [bug in Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=641509) and seems to be ok to safely ignore.

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-6.5.0
+8.2.1
```
Comment (saadaouad) [inline .node-version]: Why not use the latest stable version `8.9`?

**Comment by bahmutov** [comment]
To match the node version included in the electron version

**Pull request merged**